File: backend/src/main/java/com/team01/project/domain/notification/dto/CreateNotificationReqBody.java
package com.team01.project.domain.notification.dto;

import java.time.LocalTime;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;


public record CreateNotificationReqBody(
		@NotNull String userId,
		@NotBlank String message,
		@NotNull LocalTime notificationTime
) {
}

File: backend/src/main/java/com/team01/project/domain/notification/dto/CreateNotificationReqBody.kt
package com.team01.project.domain.notification.dto

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull
import java.time.LocalTime

data class CreateNotificationReqBody(
    @field:NotNull val userId: String,
    @field:NotBlank val message: String,
    @field:NotNull val notificationTime: LocalTime
)
